sdk/head_tracker.cc

```diff
@@ -24,8 +24,8 @@
 namespace cardboard {
 namespace {
 
-void GetViewporOrientationRotation(
-        const CardboardViewportOrientation ViewportOrientation,
+void GetRotationsFromViewportOrientation(
+        const CardboardViewportOrientation Viewport_orientation,
```

Comment:
Per Google C++ style guide, function parameters should begin with a lower case letter, please fix.

Reply:
Haha sorry, I didn't see that 'V'